README.md

```diff
@@ -100,50 +248,137 @@ This will keep the global `editor.formatOnSave` for other languages you use, and
+- A number of different folder configurations (specifically when your root module is not a parent to any submodules) are not yet supported. More information available in ([terraform-ls#32](https://github.com/hashicorp/terraform-ls/issues/32#issuecomment-649707345))
+
+### Terraform 0.11 compatibility
+
+If you are using a Terraform version prior to 0.12.0, you can install the pre-transfer 1.4.0 version of this extension by following the instructions in the [pin version section](#pin-to-a-specific-version-of-the-extension).
+
+The configuration has changed from 1.4.0 to v2.X. If you are having issues with the Language Server starting, you can reset the configuration to the following:
+
+```json
+"terraform.languageServer": {
+  "external": true,
+  "args": ["serve"]
+}
+```
+
+## Troubleshooting
+
+- If you have a question about how to accomplish something with the extension, please ask on the [Terraform Editor Discuss site](https://discuss.hashicorp.com/c/terraform-core/terraform-editor-integrations/46)
+- If you come across a problem with the extension, please file an [issue](https://github.com/hashicorp/vscode-terraform/issues/new/choose).
+- If someone has already filed an issue that encompasses your feedback, please leave a 👍/👎 reaction on the issue
+- Contributions are always welcome! Please see our [contributing guide](https://github.com/hashicorp/vscode-terraform/issues/new?assignees=&labels=enhancement&template=feature_request.md) for more details
+- If you're interested in the development of the extension, you can read about our [development process](DEVELOPMENT.md)
+
+### Generate a bug report
+
+Experience a problem? You can have VS Code open a Github issue in our repo with all the information filled out for you. Open the Command Palette and invoke `Terraform: Generate Bug Report`. This will inspect the VS Code version, the Terrafform extension version, the terraform-ls version and the list of installed extensions and open a browser window with Github loaded. You can then inspect the information provided, edit if desired, and submit the issue.
+
+### Reload the extension
+
+If you haven't see the Problems Pane update in awhile, or hover and intellisense doesn't seem to showing up, you might not know what to do. Sometimes the Terraform extension can experience problems which cause the language server to crash or not respond. The extension has a way of logging the crash, but there is something you can do to get right back to working: reload the Terraform Language Server.
+
+You can reload the Terraform Language Server by opening the command palette and starting to type Reload. A list of commands will appear, select Reload Window. This will reload the Visual Studio Code window without closing down the entire editor, and without losing any work currently open in the editor.
```

Comment:
I understand we want to enable and unblock people but I would still stress here to report the crash before reloading, esp. because the problem may not be reproducible anymore afterwards.

Crash in Go generally is pretty much 99% of the times indication of a bug. Go programs practically never crash on purpose as `panic` is not the way common errors would be handled (i.e. panic != exceptions in other languages).

Reply:
Reworked this section. I realize after re-reading, using words like crash implied an intent I was not going for.